javaslang-test/src/test/java/javaslang/test/ArbitraryTest.java

```diff
@@ -185,6 +189,79 @@ public int nextInt(int bound) {
         assertThat(actual.length()).isEqualTo(10);
     }
 
+    @Test
+    public void shouldCreateArbitraryLocalDateTime(){
+        final Arbitrary<LocalDateTime> date = Arbitrary.localDateTime();
+
+        assertThat(date).isNotNull();
+    }
+
+
+    @Test(expected = NullPointerException.class)
+    public void shouldNotAcceptNullMedianLocalDateTime(){
+        final Arbitrary<LocalDateTime> dates = Arbitrary.localDateTime(null, ChronoUnit.DAYS);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldNotAcceptNullChronoUnit(){
+        final Arbitrary<LocalDateTime> dates = Arbitrary.localDateTime(LocalDateTime.now(), null);
```

Comment:
\[FindBugs\] INFO: DLS: Dead store to dates in javaslang\.test\.ArbitraryTest\.shouldNotAcceptNullChronoUnit\(\)